adam/perception/visual_perception.py

```diff
@@ -165,25 +195,42 @@ def from_mapping(
                             weight=strokes_map["confidence_score"],
                         )
                     )
+            relative_properties = {}
+            if cluster_map["relative_distance"] is not None:
+                for other_object_name, relative_distance in cluster_map["relative_distance"].items():
+                    other_object_id = object_name_to_id(other_object_name)
+                    relative_properties[other_object_id] = []
+                    for label, value in relative_distance.items():
+                        relative_properties[other_object_id].append(ContinuousNode(label=label, value=value, weight=1.0))
+            if cluster_map["relative_size"] is not None:
+                for other_object_name, relative_size in cluster_map["relative_size"].items():
+                    other_object_id = object_name_to_id(other_object_name)
+                    for label, value in relative_size.items():
+                        # Have to convert value to `str` to fit CategoricalNode
+                        relative_properties[other_object_id].append(CategoricalNode(label=label, value=str(value), weight=1.0))
```

Comment:
The labels here will be `x_greater_than` and `y_greater_than`, correct?

Reply:
The labels here will be `height_greater_than` and `width_greater_than`. Would you prefer that I put these into a named constant (similar to the [size features](https://github.com/isi-vista/adam/blob/907afb228ffdc6df58fd2d4f68db450f244b897d/adam/perception/visual_perception.py#L35))

Reply:
Yes, that might be helpful for readability.